cookbook/controller/service.rst

```diff
@@ -271,6 +271,11 @@ controller:
 
        $router->generate($route, $params, $absolute);
 
+    .. note::
+
+        The ``$absolute`` argument must be one of the constants defined in
```

Comment:
The argument is actually named ``$referenceType`` in Symfony, not ``$absolute``, as ``$absolute`` does not make sense anymore since it is not a simple boolean anymore.
I suggest to update the doc to use the uptodate name